annotator/annotation.py

```diff
@@ -96,8 +96,28 @@ def _build_query(cls, query=None, offset=None, limit=None, sort=None, order=None
         if query is None:
             query = {}
 
+        # Pop 'before' and 'after' parameters out of the query
+        after = query.pop('after', None)
+        before = query.pop('before', None)
+
         q = super(Annotation, cls)._build_query(query, offset, limit, sort, order)
 
+        # Create range query from before and/or after
+        if before or after:
```

Comment:
It's probably okay here, but I generally try to be explicit when I want to check if something is `None`. For this case, it probably works fine, but there are other times when values are falsy and what you really mean is not `None`.